app/controllers/api/stories_controller.rb

```diff
@@ -74,11 +77,8 @@ def scoped(relation)
   end
 
   def sorted(res)
-    if filters.purchased?
-      res.purchased.order('purchase_count DESC')
-    else
-      res.order('published_at desc')
-    end
+    res = res.purchased.order('purchase_count DESC') if filters.purchased?
```

Comment:
Just to double check ... this `order` will get applied before the default sort, correct?